src/factory.js

```diff
@@ -157,6 +159,14 @@ export class Factory {
     return note;
   }
 
+  GhostNote(noteStruct) {
+    const ghostNote = new GhostNote(noteStruct);
+    if (this.stave) ghostNote.setStave(this.stave);
+    ghostNote.setContext(this.context);
+    this.renderQ.push(ghostNote);
+    return ghostNote;
```

Comment:
These four lines are duplicated all over this class. I wonder if we could replace them with something like `return this.setupElement(ghostNote)`.

No action needed here -- just a thought.